samigo/samigo-hibernate/src/java/org/sakaiproject/tool/assessment/data/dao/grading/MediaData.hbm.xml

```diff
@@ -13,7 +13,7 @@
     <many-to-one name="itemGradingData"
       class="org.sakaiproject.tool.assessment.data.dao.grading.ItemGradingData"
       column="ITEMGRADINGID" index="SAM_MEDIA_ITEMGRADING_I" />
-    <property name="media">
+    <property name="dbMedia">
```

Comment:
There is actually no database change, so no conversion script needed. This is just moving the raw database access to a new property name to make room for the smart property under the existing name. Only the DAO needs to know the difference while all other code ignores the new one.

Do you have a suggestion for some documentation? I'm not sure how we could make the implementation more clear or encapsulated.